vllm/sequence.py

```diff
@@ -466,6 +530,10 @@ class SequenceGroupMetadata:
     Args:
         request_id: The ID of the request.
         is_prompt: Whether the request is at prompt stage.
+        is_chunked_prefill: Whether the request is at chunked prefill stage.
```

Comment:
is this parameter still needed if you can figure it out from chunked prefill feature flag and `get_num_uncomputed_tokens`

Reply:
good point. I don't think we need it. Removed

Reply:
oh sorry. actually it is needed to figure out the # of chunked prefill requests within a batch